for-upstream/common.nix

```diff
@@ -97,6 +97,11 @@ in
   };
 
   config = mkIf cfg.enable {
+    nix.extraOptions = ''
+      # A store path that was missing at first may well have finished building,
+      # even shortly after the previous lookup. This *also* applies to the daemon.
+      narinfo-cache-negative-ttl = 0
```

Comment:
I think you shouldn't enforce global nix options for all your users

Reply:
Me neither, but otherwise you will encounter real evaluation errors that don't go away with a retry.
Eventually this should be replaced by explicit cache invalidation in Nix and its daemon protocol, but that's not a priority right now.

Is this setting a real problem for you?

Reply:
It's not too problematic, it's just confusing to have nix options changed